ha/k8s_setup/ha_setup.py

```diff
@@ -190,13 +182,12 @@ def process(self):
 
             conf_file_dict = {'LOG' : {'path' : const.HA_LOG_DIR, 'level' : const.HA_LOG_LEVEL},
                          'consul_config' : {'endpoint' : consul_endpoint},
-                         'prometheus_config' : {'endpoint' : None, 'poll_time': None},
                          'event_topic' : 'hare',
                          'data_pod_label' : data_pod_label,
-                         'MONITOR' : {'message_type' : 'k8s_event', 'producer_id' : 'k8s_monitor'}}
+                         'MONITOR' : {'message_type' : 'k8s_event', 'producer_id' : 'k8s_monitor'},
                          'EVENT_MANAGER' : {'message_type' : 'health_events', 'producer_id' : 'system_health',
-                                            'consumer_group' : 'health_monitor', 'consumer_id' : '1'}
-                         }
+                                            'consumer_group' : 'health_monitor', 'consumer_id' : '1'},
+                         'event_topic' : 'hare'}
```

Comment:
Will not affect functionality, but seems like "event_topic" : "hare" dictionary entry is duplicated on line 185 and 190.

Reply:
Yeah Right. Fixed this now